config/rbac/role.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+    - ""
+  resources:
+    - secrets
+  verbs:
+    - list
+    - watch
 - apiGroups:
   - operator.kyma-project.io
   resources:
```

Comment:
I think, we only need to watch, get, list eventingauth resources here as the resource is input to eventingauth controller. Hence, I'd suggest to have the following:
```yaml
resources:
  - eventingauths
verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
```